anndata/_core/raw.py

```diff
@@ -31,7 +43,10 @@ def __init__(
             self._var = _gen_dataframe(var, self.X.shape[1], ["var_names"])
             self._varm = AxisArrays(self, 1, varm)
         elif X is None:  # construct from adata
-            self._X = adata.X.copy()
+            if isinstance(adata.X, CPType.classes()):
```

Comment:
```suggestion
            # Move from GPU to CPU since it's large and not always used
            if isinstance(adata.X, (CupyArray, CupySparseMatrix)):
```